Sources/Subs-Db-mysql.php

```diff
@@ -696,8 +696,10 @@ function smf_db_error($db_string, $connection = null)
  * @param array $keys The keys for the table
  * @param bool $disable_trans Whether to disable transactions
  * @param resource $connection The connection to use (if null, $db_connection is used)
+ * @param returnmode 0 = nothing(default), 1 = last row id, 2 = all rows id as array; every mode runs only with method = ''
```

Comment:
Int would be not fully true,
from technical pov is int[] when returnmode is 2 and
when you use pg string and string[] is possible, too.

That was I mean with:

> only on ai fields -- for the second I don't see any checks

i mean we could hide this behavior if we that don't mention ...